cql3/cql_statement.hh

```diff
@@ -118,6 +119,23 @@ public:
     virtual bool is_conditional() const {
         return false;
     }
+
+    enum class cql_statement_type {
+        NO_METADATA,
+        OPT_METADATA,
+        SELECT
+    };
```

Comment:
There are more CQL statement types than those (and `NO_METADATA` and `OPT_METADATA` are not real ones).